--- origsrc/libpagemaker-0.0.2/src/lib/Makefile.am	2014-06-20 08:36:55.000000000 -0500
+++ src/libpagemaker-0.0.2/src/lib/Makefile.am	2015-04-21 17:00:12.270944400 -0500
@@ -14,7 +14,7 @@ AM_CXXFLAGS = -I$(top_srcdir)/inc $(REVE
 
 libpagemaker_@PMD_MAJOR_VERSION@_@PMD_MINOR_VERSION@_la_LIBADD  = $(REVENGE_LIBS) @LIBPMD_WIN32_RESOURCE@
 libpagemaker_@PMD_MAJOR_VERSION@_@PMD_MINOR_VERSION@_la_DEPENDENCIES = @LIBPMD_WIN32_RESOURCE@
-libpagemaker_@PMD_MAJOR_VERSION@_@PMD_MINOR_VERSION@_la_LDFLAGS = $(version_info) -export-dynamic -no-undefines
+libpagemaker_@PMD_MAJOR_VERSION@_@PMD_MINOR_VERSION@_la_LDFLAGS = $(version_info) -export-dynamic -no-undefined
 libpagemaker_@PMD_MAJOR_VERSION@_@PMD_MINOR_VERSION@_la_SOURCES = \
 	OutputShape.cpp \
 	OutputShape.h \
